specs/AI-034-opencode-npm-channel/verification.md

```
@@ -0,0 +1,67 @@
+created: "2026-08-27"
+---
+
+# Verification - AI-034-opencode-npm-channel
+
+## Evidence
+
+All produced 2026-08-27 on the Windows work box (where the defect was measured), from the `feat/opencode-npm-channel` worktree.
+
+`features.json` verification commands (Git Bash, Go 1.26, jq, bats 1.13):
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)